src/stylesheets/components/_banner.scss

```diff
@@ -159,6 +163,14 @@ $banner-icon-close: (
   @include at-media("tablet") {
     display: none;
   }
+
+  @media (forced-colors: active) {
+    color: LinkText;
+
+    &::after {
+      background-color: ButtonText;
+    }
+  }
```

Comment:
# Banner

The banner was missing chevrons, which implies there is content hidden behind the link. It was also missing the close icon on mobile which allows users to close the pane.

The icons were already placed as masks using `place-icon()`, so the solution was simply setting the background color to an appropriate System Color

_Current High Contrast / High Contrast FIx_
![image](https://user-images.githubusercontent.com/25211650/159998865-ba178e91-c6c0-4a66-a942-0e900446303e.png)